### DISCUSSION TITLE
DNS Set up. I have 3 SPF records - is that correct?

### DISCUSSION THREAD

**Discussion by Dave777** [postalserver/postal#2723] [Help with using Postal]
HI,
I followed the advice here:
https://docs.postalserver.io/install/dns-configuration/

and I have spf for the root domain and  2 sub domains so 3 SPF records

Is this set up correct ?

Many thanks.

**Comment by willpower232** [comment]
As long as they aren't all on the same subdomain, it is fine in general terms. I don't think there are any problems with that you have screenshotted.
